src/lib/Strategy/FocusMode/ContentStructureRedirectStrategy.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Strategy\FocusMode;
+
+use Ibexa\Contracts\AdminUi\FocusMode\RedirectStrategyInterface;
+use Ibexa\Contracts\Core\Repository\LocationService;
+use Ibexa\Contracts\Core\SiteAccess\ConfigResolverInterface;
+use Symfony\Component\Routing\RouterInterface;
+
+final class ContentStructureRedirectStrategy implements RedirectStrategyInterface
```

Comment:
I'd make this implementation internal. Unless it was left like that on purpose :)